defaults/install/the-build/build.yml

```diff
@@ -53,13 +53,6 @@ drupal:
 behat:
   args: "--suite=default --strict"
 
-# Configuration for checking the site with the Drupal Checker code linter.
```

Comment:
@becw Same goes for this -- would we not want this to run when building? I'm probably just misunderstanding :P

Reply:
Again, it does still run -- but we don't need to set an environment-specific value with config anymore because we know it's going to be at `vendor/bin/drupal-check`.